sourcegraph/sourcegraph.proto

```diff
@@ -808,6 +808,8 @@ service Changesets {
 	// update event. If no update occurred, it returns nil.
 	rpc Update(ChangesetUpdateOp) returns (ChangesetEvent);
 
+	rpc Merge(ChangesetMergeOp) returns (pbtypes.Void);
```

Comment:
My style guide and consistency advisor may complain that this is the only method within `Changesets` service that isn't documented.